src/views/common/TimeScrollView/TimeAxisTicks.ts

```diff
@@ -0,0 +1,122 @@
+import { matrix, Matrix, multiply } from 'mathjs';
```

Comment:
This is largely just broken out unchanged from the original TimeScrollView. I did update to use the new fully-encapsulated pixel conversion.